plugins/modules/system/homectl.py

```diff
@@ -0,0 +1,618 @@
+        type: str
+    umask:
+        description:
+            - Sets the umask for the user's login sessions
+            - Value from C(0000) to C(0777).
+        type: int
+    memberof:
+        description:
+            - String separated by comma each indicating a UNIX group this user shall be a member of.
+            - Groups the user should be a member of should be supplied as comma separated list.
+        aliases: [ 'groups' ]
+        type: str
+    skeleton:
+        description:
+            - The absolute path to the skeleton directory to populate a new home directory from.
+            - This is only used when a home directory is first created.
+            - If not specified homed by default uses ``/etc/skel``
+        aliases: [ 'skel' ]
+        type: path
+    shell:
+        description:
+            - Shell binary to use for terminal logins of given user.
+            - If not specified homed by default uses ``/bin/bash``
+        type: str
+    environment:
+        description:
+            - String separated by comma each containing an environment variable and its value to
+              set for the user's login session, in a format compatible with ``putenv()``.
+            - Any environment variable listed here is automatically set by pam_systemd for all
+              login sessions of the user.
+        aliases: [ 'setenv' ]
+        type: str
+    timezone:
+        description:
+            - Preferred timezone to use for the user.
+            - Should be a tzdata compatible location string such as C(America/New_York).
+        type: str
+    locked:
+        description:
+            - Whether the user account should be locked or not.
+        type: bool
+    language:
+        description:
+            - The preferred language/locale for the user.
+            - This should be in a format compatible with the C($LANG) environment variable.
+        type: str
+    passwordhint:
+        description:
+            - Password hint for the given user.
+        type: str
+    sshkeys:
+        description:
+            - String separated by comma each listing a SSH public key that is authorized to access the account.
+            - The keys should follow the same format as the lines in a traditional ``~/.ssh/authorized_key`` file.
+        type: str
+    notbefore:
+        description:
+            - A time since the UNIX epoch before which the record should be considered invalid for the purpose of logging in.
+        type: int
+    notafter:
+        description:
+            - A time since the UNIX epoch after which the record should be considered invalid for the purpose of logging in.
+        type: int
+'''
+
+EXAMPLES = '''
+- name: Add the user 'james'
+  community.general.homectl:
+    name: johnd
+    password: myreallysecurepassword1!
+    state: present
+
+- name: Add the user 'tom' with a zsh shell, uid of 1000, and gid of 2000
+  community.general.homectl:
+    name: tom
```

Comment:
How about also using some female names in the example?
```suggestion
- name: Add the user 'alice' with a zsh shell, uid of 1000, and gid of 2000
  community.general.homectl:
    name: alice
```

Reply:
Agreed! Great Idea.